lib/integration/scenarios/Test/Integration/Run.hs

```diff
@@ -97,9 +106,10 @@ main = withTestsSetup $ \testDir (tr, tracers) -> do
         _noConway = ignoreInConway localClusterEra
         _noBabbage = ignoreInBabbage localClusterEra
         testnetMagic = Cluster.TestnetMagic (natVal (Proxy @netId))
-    testDataDir <-
-        FileOf . fromMaybe "."
+    testDataDir <- do
+        dir <- fromMaybe "."
             <$> lookupEnv "CARDANO_WALLET_TEST_DATA"
+        DirOf . absDir <$> makeAbsolute dir
```

Comment:
```suggestion
        DirOf <$> absolutize dir
```

Use `absolutize` here — in order to be consistent with the treatment of the other environment variables?

Reply:
Yes, overlooked